pkg/api/validation/validation.go

```diff
@@ -2346,6 +2356,7 @@ func ValidateResourceRequirements(requirements *api.ResourceRequirements, fldPat
 		if api.IsStandardResourceName(string(resourceName)) {
 			allErrs = append(allErrs, validateBasicResource(quantity, fldPath.Key(string(resourceName)))...)
 		}
+		allErrs = append(allErrs, validateContainerResourceLimits(string(resourceName), quantity, fldPath)...)
```

Comment:
@derekwaynecarr: Are you suggesting disabling quota if limits < 10m?
I can add validation to limit ranges in this PR. Good point.